listenbrainz/webserver/views/art_api.py

```diff
@@ -101,28 +107,61 @@ def cover_art_grid_post():
     if err is not None:
         raise APIBadRequest(err)
 
-    if not isinstance(r["release_mbids"], list):
-        raise APIBadRequest("release_mbids must be a list of strings specifying release_mbids")
-
-    for mbid in r["release_mbids"]:
-        if not is_valid_uuid(mbid):
-            raise APIBadRequest(f"Invalid release_mbid {mbid} specified.")
-
     if "cover_art_size" in r:
         cover_art_size = r["cover_art_size"]
     elif r["image_size"] < 1000:
         cover_art_size = 250
     else:
         cover_art_size = 500
 
-    images = cac.load_images(r["release_mbids"], tile_addrs=tiles, layout=layout, cover_art_size=cover_art_size)
+    # Get release_mbids or release_group_mbids
+    mbids = []
+    entity = "release"
+    
+    def look_for_release_group_mbids():
+        if "release_group_mbids" not in r:
+            return
+        
+        if not isinstance(r["release_group_mbids"], list):
+            raise APIBadRequest("release_group_mbids must be a list of strings specifying release_group_mbids")
+
+        nonlocal mbids 
```

Comment:
The use of nonlocal to return values to the calling function is weird. Why not pass in the values and return them at the end of the function?

Reply:
I usually like this if its an inner function since the scope of an inner function's usage is very limited, meaning there are no ghost usages (usages that devs aren't aware about). I would not have to create new variables again and again, but rather just assign the variables their respective values without much boilerplate, hence looks nicer syntactically. 

Thats just my pov, totally your call on how we should go forward.

Reply:
Yeah, let's avoid nonlocal. We don't use it anywhere else in the code and things like ghost usages are not an issue because this is all internal code, no library or direct external use intended. You can prefix the function with an `_` to hint that it is private.

Reply:
Done